homeassistant/components/nut/const.py

```diff
@@ -7,14 +7,14 @@
     DEVICE_CLASS_VOLTAGE,
 )
 from homeassistant.const import (
-    ELECTRICAL_CURRENT_AMPERE,
-    ELECTRICAL_VOLT_AMPERE,
+    ELECTRIC_CURRENT_AMPERE,
+    ELECTRIC_POTENTIAL_VOLT,
     FREQUENCY_HERTZ,
     PERCENTAGE,
+    POWER_VOLT_AMPERE,
```

Comment:
Volt Ampere is the same as Watt. 
So why should we not always use Watt?

Reply:
Ok, I read this: 
> They are, however, sometimes used in different contexts so it's worth to keep both.

Reply:
If you are interested, this should explain it
https://www.electronicdesign.com/markets/energy/article/21801657/whats-the-difference-between-watts-and-voltamperes

Tbh I was a bit imprecise when I said `1 VA = 1 W`. That is only true in certain applications.